pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -116,6 +131,51 @@ def configure(self,
             folder, filename,
             output_titles)
 
+        self.status_output = False
+        self.infectiousness_output = False
+        self.ih_status_writer = None
+        self.ih_infectiousness_writer = None
+
+        if ih_file_params:
+            # Setting up writer for infection history for each person. If the
+            # ih_file_params dict is empty, then we do not need to record this
+            self.status_output = ih_file_params["status_output"] \
+                if "status_output" in ih_file_params else False
+
+            self.infectiousness_output = \
+                ih_file_params["infectiousness_output"] \
+                if "infectiousness_output" in ih_file_params else False
```

Comment:
Seems like you want the [`get`](https://python-reference.readthedocs.io/en/latest/docs/dict/get.html) method here